docs/source/api/deliveryservices_xmlid_xmlid_sslkeys.rst

```diff
@@ -71,9 +71,13 @@ Response Structure
 :hostname:        The hostname generated by Traffic Ops that is used as the common name when generating the certificate - this will be a FQDN for DNS :term:`Delivery Services` and a wildcard URL for HTTP :term:`Delivery Services`
 :organization:    An optional field which, if present, contains the organization entered by the user when generating certificate\ [1]_
 :state:           An optional field which, if present, contains the state entered by the user when generating certificate\ [1]_
-:version:         The version of the certificate record in Riak
+:version:         An integer that defines the "version" of the key - which may be thought of as the sequential generation; that is, the higher the number the more recent the key
 
-.. code- block:: http
+:expiration:      The expiration date of the certificate for the :term:`Delivery Service` following RFC3339 internet date/time format YYYY-MM-DDTHH:mm:ssZ
```

Comment:
Same as above RE: optionally linking to the RFC (just don't want you to miss one in case you choose to change it)

Reply:
done